src/player_display.cpp

```diff
@@ -677,60 +677,56 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4" ) );
                 if( line == 0 ) {
                     // Display player current strength effects
                     mvwprintz( w_stats, 2, 1, h_light_gray, _( "Strength:" ) );
-                    mvwprintz( w_stats, 6, 1, c_magenta, _( "Base HP:" ) );
```

Comment:
Comments about current effects from stats should be moved\removed, because these effects described in another place now.

Reply:
Thanks for catching that, I've added a commit which clarifies the comments in question.